scripts/variantstore/wdl/extract/hail_gvs_import.py

```diff
@@ -24,6 +24,7 @@
 
 def import_gvs(argsfn, vds_path, references_path, temp_path):
```

Comment:
see my comment above about potentially renaming this function from import_gvs to something else.  It is not strictly necessary, but if it were to be done this is the ticket that we were going to do it in, I believe